pad_crm/README.rst

```diff
@@ -0,0 +1,44 @@
+=============
+ Pad support
+=============
+
+This module adds a Pad to the lead in CRM along with collaborative editing of internal notes and maintains the versions history.
```

Comment:
``adds a Pad ... along with ...`` -- not exactly. Without module, there is a **Description** field, but it lacks html support and colloborative editiing. So, *pad* **is**  collaborative editing + formatting support. IMO, something like *Adds collobartive editing with html support in Lead's Internal notes* is more appropriate

Reply:
fixed